tools/update_runnablejobs.py

```diff
@@ -0,0 +1,110 @@
+import os
```

Comment:
I move the whole update runnablejobs into a new file because it's a little complex now and we really need to test it. :)